app/log/config.go

```
@@ -72,101 +80,175 @@ func InitLogger(config Config) error {
 		return err
 	}
 
+	writer, _, _ := zap.Open("stderr")
```

why ignoring this error?

---

copy/paste, thanks fixed